Model/User.php

```diff
@@ -120,7 +120,6 @@
      */
     public function __construct()
     {
-        $this->salt = base_convert(sha1(uniqid(mt_rand(), true)), 16, 36);
```

Comment:
Covering what ? the code is gone, as it is useless to generate the salt on creation now (as we will replace it anyway)